examples/bridge-app/bridge-common/bridge-app.zap

```diff
@@ -5288,9 +5288,123 @@
           ]
         }
       ]
+    },
+    {
+      "name": "Anonymous Endpoint Type",
```

Comment:
are you able to specify explicitly an endpoint id or does it just compute based upon the array index?

Reply:
> are you able to specify explicitly an endpoint id or does it just compute based upon the array index?

We are able to specify the `endpointId` explicitly in the zap tool. There is also a `endpointTypeIndex` in the zap file which gets generated automatically.